react/src/components/ServingRouteErrorModal.tsx

```diff
@@ -22,7 +17,7 @@ const ServingRouteErrorModal: React.FC<Props> = ({
   onRequestClose,
   onCancel,
   inferenceSessionErrorFrgmt,
-  ...modalProps
+  ...props
```

Comment:
How about changing this to "baiModalProps"? It has only BAIModalProps. Let's make it clear. 

If you agree, please change the other files as well.

Reply:
reflected!